gdx/src/com/badlogic/gdx/math/Vector2.java

```diff
@@ -351,6 +351,14 @@ public float angleDeg (Vector2 reference) {
 		return angle;
 	}
 
+	/** @return the angle in degrees of this vector (point) relative to the given vector. Angles are towards the positive y-axis
+	 *         (typically counter-clockwise.) in the [0, 360) range */
+	public static float angleDeg (float x, float y) {
```

Comment:
This has some utility, but I'd rather see variants on MathUtils.atan2 that return degrees. I've already written them, anyway. https://github.com/tommyettinger/digital/blob/d00ba5bc961bcab8b0a05dc4a2401ce165856250/src/main/java/com/github/tommyettinger/digital/TrigTools.java#L316-L375

Since MathUtils is almost all static methods, this type of code makes sense there; I don't want there to be a case where it isn't clear if a static or member method should be used on Vector2. The argument order does make more sense here than with the traditional `atan2(y, x)`.

Reply:
What about the other already implemented static functions in `Vector2 ` should they be moved, too, then?
```
public static float len (float x, float y)
public static float len2 (float x, float y)
public static float dot (float x1, float y1, float x2, float y2)
public static float dst (float x1, float y1, float x2, float y2)
public static float dst2 (float x1, float y1, float x2, float y2)
```

Regarding the new `public static float angleDeg (float x, float y)` function, it just copies the same code from the other two non-static `angleDeg` implementations in `Vector2` (it is done the same way for  the above static methods, so I thought I'd keep it consistent). Should their existing code be changed then as well?

I'm confused what to do now.

And 'Im sorry, I disagree about the argument order as everthing else in `Vector2` has `(float x, float y)` in this order and bugs will happen because nobody expects a single function the have another order than all the rest... This seems pretty counter-intuitive to me.

Reply:
No, nothing should be moved, that would break existing code.

Honestly I don't use Vector2 much because I mostly work with grid-based code, so I haven't had to dig into its code too much. I feel like Vector2 in libGDX should call MathUtils in libGDX when possible, as opposed to using Math and casting back and forth.

As for argument order, (x, y) makes more sense than (y, x), it's just that atan2 traditionally uses y first.

Reply:
This could just call `MathUtils.atan2Deg360(float y, float x)` now, and be a one-liner. It would be less precise, though, and since this code seems to be using Math where possible, I guess we're fine as-is.